static/js/recent_topics.js

```diff
@@ -29,6 +29,9 @@ let col_focus = 1;
 // increased when we add new actions, or rethought if we add optional
 // actions that only appear in some rows.
 const MAX_SELECTABLE_COLS = 4;
+// We don't set a current_focus_element unless we are sure that
+// user is using keyboard to navigate.
+let user_uses_keyboard = false;
```

Comment:
Can you explain this change in more detail?  I don't understand its purpose.

Reply:
Yeah, If user is not using keyboard to navigate we don't need to call `revive_current_focus` every time we update the view and highlight the focused element. I thought it would stop this unnecessary call for us in case user is just using mouse, I can remove it if it seems odd.